app/views/about/accessibility.en.html.erb

```diff
@@ -0,0 +1,84 @@
+
+        <p class="govuk-body">This online tool is partially compliant with the <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/" rel="external" target="_blank">Web Content Accessibility Guidelines version</a> 2.1 AA standard, due to the non-compliances listed below.</p>
+
+        <h2 class="govuk-heading-m">Non accessible content</h2>
+
+        <p class="govuk-body">The content listed below is non-accessible for the following reasons.</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>Third-party supplied feedback form</li>
+        </ul>
+
+        <h2 class="govuk-heading-m">Non compliance with the accessibility regulations</h2>
+
+        <p class="govuk-body">This online tool uses a third-party supplied service to collect feedback.</p>
+
+        <p class="govuk-body">This third-party supplied service is ‘skinned' to look like our website, but some parts of it do not meet <a class="govuk-link" href="https://www.w3.org/TR/WCAG21/" rel="external" target="_blank">Web Content Accessibility Guidelines version 2.1</a> standards because:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>incorrect heading tags are used</li>
+          <li>clicking the labels does not move the cursor into the corresponding text area</li>
+          <li>when using the keyboard, the radio button becomes invisible</li>
+        </ul>
+
+        <p class="govuk-body">The third-party supplied feedback form is one part of our overall approach to collecting feedback, including face-to-face interviews carried out with users in testing. We'll continue to carry out further user testing as we develop the service. We plan to reassess the feedback form by April 2020, before the full version of this online tool is live.</p>
+
+        <h2 class="govuk-heading-m">Reporting accessibility problems</h2>
+
+        <p class="govuk-body">We're always looking to improve the accessibility of this online tool. If you find any problems not listed on this page or you think we're not meeting accessibility requirements, contact <a href="mailto:disclosure.checker@digital.justice.gov.uk">disclosure.checker@digital.justice.gov.uk</a>.</p>
+
+        <h2 class="govuk-heading-m">Enforcement procedure</h2>
+
+        <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations'). If you're not happy with how we respond to your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/" rel="external" target="_blank">contact the Equality Advisory and Support Service (EASS)</a>.</p>
+
+        <p class="govuk-body">This statement was prepared on 24 October 2019 It was last updated on 24 October 2019.</p>
```

Comment:
I think a full stop or a new line will be nice here, I've added a comment in Wes' document although is a minor thing.